config/routes.rb

```diff
@@ -526,7 +526,15 @@ def matches?(request)
   end
   ####
 
+  #### Admin::Debug controller
+  namespace :admin do
+    # FIXME: For some reason the resources call is generating the route as
+    # admin_debug_index_path rather than the standard admin_debug_path.
+    # resources :debug, only: [:index]
```

Comment:
This is to be expected when using the singlar form of a resource. Obviously we don't want route to be `/admin/debugs` tho, I would be fine with having to use `admin_debug_index_path`.